sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1226,6 +1226,22 @@ static void _init_from_device(gnrc_netif_t *netif)
     _update_l2addr_from_dev(netif);
 }
 
+static void _configure_netdev(netdev_t *dev)
+{
+    /* Enable RX- and TX-complete interrupts */
+    static const netopt_enable_t enable = NETOPT_ENABLE;
+    int res = dev->driver->set(dev, NETOPT_RX_END_IRQ, &enable, sizeof(enable));
+    if (res < 0) {
+        LOG_ERROR("gnrc_netif: enable NETOPT_RX_END_IRQ failed: %d\n", res);
```

Comment:
Couldn't that have been `DEBUG`? I now get this error message every time on `native`.

Reply:
See #9525

Reply:
You're right, DEBUG is probably better